pkg/render/fluentd.go

```diff
@@ -1040,10 +1042,38 @@ func (c *fluentdComponent) eksLogForwarderDeployment() *appsv1.Deployment {
 		{Name: "AWS_REGION", Value: c.cfg.EKSConfig.AwsRegion},
 		{Name: "AWS_ACCESS_KEY_ID", ValueFrom: secret.GetEnvVarSource(EksLogForwarderSecret, EksLogForwarderAwsId, false)},
 		{Name: "AWS_SECRET_ACCESS_KEY", ValueFrom: secret.GetEnvVarSource(EksLogForwarderSecret, EksLogForwarderAwsKey, false)},
+		relasticsearch.ElasticIndexSuffixEnvVar(c.cfg.ESClusterConfig.ClusterName()),
+		relasticsearch.ElasticUserEnvVar(ElasticsearchEksLogForwarderUserSecret),
+		relasticsearch.ElasticPasswordEnvVar(ElasticsearchEksLogForwarderUserSecret),
+		relasticsearch.ElasticSchemeEnvVar(esScheme),
+		relasticsearch.ElasticHostEnvVar(esHost),
```

Comment:
One to keep an eye on - once @vara2504 is done with her work for EKS log forwarder, we can remove most (all?) of these.